web/test/test_persist.py

```diff
@@ -37,7 +37,7 @@ def test_requirement_satisfied(self):
 
     @pytest.mark.skipif(is_dev_env(), reason="Not running in build environment.")    
     def test_update_from_scrape(self):
-        faculty = Faculty(name="William.Allison", faculty_id=379, email="william.allison@ualberta.ca")
+        faculty = Faculty(name="William.Allison", full_name="Allison, William.", faculty_id=379, email="william.allison@ualberta.ca")
```

Comment:
faculty type has to have full name now too? or this is just an example?

Reply:
It's now a required field. We need it to match nserc grants with profs